<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Kommentar</class>
 <widget class="QDialog" name="Dialog_Kommentar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kommentar</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>191</x>
     <y>90</y>
     <width>301</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_Abbrechen">
      <property name="text">
       <string>Abbrechen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_save">
      <property name="text">
       <string>Speichern</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_OK">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>10</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_Kommentar">
        <property name="text">
         <string>Kommentar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_AFB">
        <property name="text">
         <string>Ausf√ºhrbedingung</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_Kommentar_var">
        <property name="text">
         <string>[KOM]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_AFB_var">
        <property name="text">
         <string>[AFB]</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit_Kommentar"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_AFB"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_Kommentar</tabstop>
  <tabstop>lineEdit_AFB</tabstop>
  <tabstop>pushButton_Abbrechen</tabstop>
  <tabstop>pushButton_save</tabstop>
  <tabstop>pushButton_OK</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
